invenio_logging/sentry.py

```diff
@@ -35,6 +39,13 @@ def init_app(self, app):
         if app.config["SENTRY_DSN"] is None:
             return
 
+        # Check if sentry-sdk is installed
+        if sentry_sdk is None:
+            app.logger.warning(
+                "Please install sentry-sdk to use the Sentry logging extension."
+            )
```

Comment:
I was considering raising a `RuntimeError` instead, but I feel it's a bit too "instrusive"